### ISSUE TITLE
remove environment block

### ISSUE THREAD

**Issue by charlierudolph** [Originate/exosphere#837]
```yml
# Before
environment:
  default:
  local:
  remote:
  secrets:

# After
# default is removed in favor of using yamls way to share code

local:
  environment:

remote:
  environment:
  secrets:
```

@alexdavid @hugobho please review.  Extracted from https://github.com/Originate/exosphere/issues/429#issuecomment-332694981

**Comment by hugoesthere** [comment]
This should be
```
local:
  environment:
  secrets:

remote:
  environment:
  secrets:
```
to accommodate for local secrets now right?

**Comment by charlierudolph** [comment]
yep

**Comment by alexdavid** [comment]
This is going to be so much better 👍

**Comment by charlierudolph** [comment]
Fixed in 2e6cc954c40983a1acaefd678cad2645949308e6
